flutter_reach_five/lib/models/consent.dart

```diff
@@ -12,7 +12,9 @@ class Consent extends Equatable {
   /// The consent granted
   final bool granted;
 
-  /// The date the consent was granted TODO: confirm that and try to use DateTime with good conversion
+  /// The date the consent was last updated
```

Comment:
Could you make it a `DateTime` and absorb the `toIso8601String()` in the reachFive dart sdk ?